features/collections/collection_invite.feature

```diff
@@ -86,3 +86,12 @@ Feature: Collection
     And 0 emails should be delivered
   When I view the approved collection items page for "anon hidden collection"
   Then I should not see "A Death in Hong Kong"
+
+  Scenario: A work with too many tags can be invited to a collection
+    Given the user-defined tag limit is 5
+      And the collection "Favorites"
+      And the work "Over the Limit"
+      And the work "Over the Limit" has 6 fandom tags
```

Comment:
Can we make this test even smaller? 2 fandom tags over a limit of 1.

Reply:
The default step for creating a work fills in 1 fandom (`Stargate SG-1`) and 1 freeform (`Scary tag`), so a limit of 1 would break that step. I might be able to do a limit of 2, but I did 5 here because I did 5 with 6 fandom tags in several other tests
https://github.com/otwcode/otwarchive/blob/ccb7f351e1736e120ccd2d092ee8fb1749e4217d/features/bookmarks/bookmark_create.feature#L433-L450
https://github.com/otwcode/otwarchive/blob/ccb7f351e1736e120ccd2d092ee8fb1749e4217d/features/collections/collection_anonymity.feature#L536-L545
https://github.com/otwcode/otwarchive/blob/ccb7f351e1736e120ccd2d092ee8fb1749e4217d/features/works/work_delete.feature#L195-L201
Do you want me to change all of them?

Reply:
Please and thanks!